runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenPlugin.java

```diff
@@ -62,6 +62,8 @@
 
 	private String usernameCache;
 
+	private static final int MAX_EMAIL_LENGTH = 254, MAX_PASSWORD_LENGTH = 20;
```

Comment:
And constants go at top of the file, before instance fields

Reply:
Oh, and also, MAX_EMAIL_LENGTH is wrong, because its username/email (for older accounts), so I would just name this username. And, declare the constants on separate lines

Reply:
I think `MAX_LOGIN_LENGTH` would probably cover email and username.